src/python/WMCore/ReqMgr/Service/Request.py

```diff
@@ -207,51 +208,27 @@ def validate(self, apiobj, method, api, param, safe):
                 msg = str(ex)
             raise InvalidSpecParameterValue(msg)
 
-    def _maskTaskStepChain(self, masked_dict, req_dict, chain_name, mask_key):
-
-        mask_exist = False
-        num_loop = req_dict["%sChain" % chain_name]
-        for i in range(num_loop):
-            if mask_key in req_dict["%s%s" % (chain_name, i + 1)]:
-                mask_exist = True
-                break
-        if mask_exist:
-            defaultValue = masked_dict[mask_key]
-            masked_dict[mask_key] = []
-            # assume mask_key is list if the condition doesn't meet.
-
-            for i in range(num_loop):
-                chain_key = "%s%s" % (chain_name, i + 1)
-                chain = req_dict[chain_key]
-                if mask_key in chain:
-                    masked_dict[mask_key].append(chain[mask_key])
-                else:
-                    if isinstance(defaultValue, dict):
-                        value = defaultValue.get(chain_key, None)
-                    else:
-                        value = defaultValue
-
-                    if value is not None:
-                        masked_dict[mask_key].append(value)
-
-            masked_dict[mask_key] = list(set(masked_dict[mask_key]))
-        return
-
     def _mask_result(self, mask, result):
+        """
+        If a mask of parameters was provided in the query string, then filter
+        the request key/values accordingly.
+        :param mask: a list of strings (keys of the request dictionary)
+        :param result: a dict key'ed by the request name, with the whole
+            request dictionary as a value
+        :return: updates the result object in place and returns it (dict)
+        """
 
         if len(mask) == 1 and mask[0] == "DAS":
             mask = ReqMgrConfigDataCache.getConfig("DAS_RESULT_FILTER")["filter_list"]
 
         if len(mask) > 0:
             masked_result = {}
             for req_name, req_info in result.items():
```

Comment:
Following the good trend from the rest of this PR of migrating from snake_case to camelCase names, I would suggest to take the opportunity and do it fro every variable in this function + the function name itself, provided we know all the places this one is called from.

Reply:
Don't you think it's kind of pointless to convert variables to camel case only in one function?
I fully agree with converting the whole module to be conformant with our pylintrc standards, but that would require a whole new validation to make sure no mistakes were made.

Reply:
I wouldn't dare to suggest a change for the whole module, but for sure I agree with you it is worth a whole separate issue.